reference_designs/horizontal_diffusion/HDIFF_dual_AIE_objectFIFO/Makefile

```diff
@@ -0,0 +1,37 @@
+# (c) 2023 SAFARI Research Group at ETH Zurich, Gagandeep Singh, D-ITET   
+  
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+ACDC_AIE = $(dir $(shell which aie-opt))/..
+ACDC_AIR = $(dir $(shell which air-opt))/..
+SYSROOT = /mnt/scratch/gagsingh/sysroot/sysroot
+
+i32: i32_chess i32.elf
+f32: f32_chess f32.elf
+
+build: 
+	xchessmk test.prx
+sim: 
+	xca_udm_dbg -P /mnt/scratch/tools_tmp/Xilinx/Vitis/2021.2/aietools/data/cervino/lib/ -t sim.tcl
+
+
+i32_chess: 
+	/mnt/scratch/tools_tmp/Xilinx/Vitis/2021.2/aietools/bin/xchesscc -p me -P /mnt/scratch/tools_tmp/Xilinx/Vitis/2021.2/aietools/data/cervino/lib/ -L /mnt/scratch/tools_tmp/Xilinx/Vitis/2021.2/aietools/data/cervino/lib/ -c ./hdiff_lap.cc ./hdiff_flux.cc
```

Comment:
Remove hard coded paths to your environment. Use the makefile variable (Line 7 and 8 instead). Apply this to all Makefiles. Thanks